src/meta/src/lib.rs

```diff
@@ -246,15 +248,15 @@ pub fn start(opts: MetaNodeOpts) -> Pin<Box<dyn Future<Output = ()> + Send>> {
                     .periodic_ttl_reclaim_compaction_interval_sec,
             },
             SystemParams {
-                barrier_interval_ms: Some(config.streaming.barrier_interval_ms),
-                checkpoint_frequency: Some(config.streaming.checkpoint_frequency as u64),
-                sstable_size_mb: Some(config.storage.sstable_size_mb),
-                block_size_kb: Some(config.storage.block_size_kb),
-                bloom_false_positive: Some(config.storage.bloom_false_positive),
+                barrier_interval_ms: Some(opts.barrier_interval_ms),
```

Comment:
Seems we will extract initial system parameters only from the CLI now. 🤔 Is this the expected behavior? By doing this, we are not able to customize them in the config file and I guess that's why this change leads to end-to-end tests timeout. cc @fuyufjh 

These configuration items do not take effect and we fallback to the default value (like 1000ms, 10 freq) which will waste time on CI.

https://github.com/risingwavelabs/risingwave/blob/632423ae6b18ffb8284d15466f3ba454c3bf42a3/src/config/ci.toml#L6-L10

Reply:
It seems that we didn't have discussion on this part. Previously, we had decided the `storage_url` can be initialized via command line, which is required by a new cluster.

But for this case, is there some better way to SET these parameters (`barrier_interval_ms` majorly) for CI?

cc. @Gun9niR

Reply:
Note that we have no guarantee about when these new values of mutable params will take effect after `SET`. So if we decide to initialize them with `SET`, we'd better restart the cluster before running tests in case. This seems a little bit complicated for testing purpose. 😟

Reply:
Hmmmm, from my understanding the `SET` may not take effect immediately but will eventually propogate to all nodes. Am I correct?

Reply:
It depends on the implementation. Sometimes for simplicity, a module initializes itself by directly snapshotting the params (like copying an `u32`), instead of subscribing to the changes. In this case, the param only takes effect for new "session"s. If a session has the same lifetime as the node, then it's on the next startup.

But indeed, we can carefully check all implementations and apply the changes ASAP.

Reply:
> Note that we have no guarantee about when these new values of mutable params will take effect after SET.

That's why I also make mutable params like `barrier_interval_ms` acceptable from CLI.